.appveyor.yml

```diff
@@ -10,12 +10,12 @@ branches:
   only:
     - master
     - develop
+    - tensor
```

Comment:
This file shouldn't contain any changes, other than additional build variants for C++17 - specific builds.

Reply:
I have the same comments as for .travis.yml. We need to clarify this point.

Reply:
The tensor branch is taken away. Still need to add build variants.